PRODUCTIONISATION.md

```diff
@@ -64,11 +64,10 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 
 * 5.1 Continuous Delivery pipeline configured to:
   * [ ] 5.1.1 Automatically test pull requests
-  * [ ] 5.1.2 `master` branch to run linting and tests
+  * [ ] 5.1.2 `master` branch to run tests
   * [ ] 5.1.3 Automatic deploy to `staging` environment
   * [ ] 5.1.4 Manual step to deploy to `production` environment
-  * [ ] 5.1.5 Zero-downtime deployment step to production
-  * [ ] 5.1.6 Manual step to flip between blue/green environment
+   * [ ] 5.1.5 Zero downtime (e.g. blue/green environment) deployment step to production
```

Comment:
Zero downtime deploys should be done for all environments, not just production.

If a staging deploy is to cause downtime, we know we shouldn't deploy to production yet

Reply:
Resolved in https://github.com/madetech/productionisation/pull/51/commits/9ab5dcd221ff276d9eead6eb1b06e1e8fc969456.